src/x509/clu_request_setup.c

```diff
@@ -73,7 +73,7 @@ static int _wolfSSL_X509_print_serial(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
 
     XMEMSET(serial, 0, sz);
     if (wolfSSL_X509_get_serial_number(x509, serial, &sz) == WOLFSSL_SUCCESS) {
-        if (sz > 1) {
+        if (sz > 0) {
```

Comment:
Check for edge case here when using a serial of 0. It may missmatch from interoperability with OpenSSL when having sz > 0 versus sz > 1.

Reply:
Restored, but even when using serial of 0, wouldn't sz>0 still be true?

Reply:
if I remember right wolfSSL is counting the 0 as 1 byte so it returns a size of 1 for it

maybe not in this PR but could you add a check around that edge case? With printing out a 0 serial value

Reply:
Yes, I can put up a separate PR

Reply:
Done here https://github.com/wolfSSL/wolfCLU/pull/61